app/components/molecules/promotion-card/promotion-card.styles.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export interface PromotionCardProps {
+  variant?: 'default' | 'recommendation' | 'seasonal';
+  bgImage?: string;
+  label?: string;
+}
+
+export const Img = styled.img``;
```

Comment:
What's the purpose of this empty component?

Reply:
Literally nothing. I created it in case I would need to position the image and forgot to remove it when I realized I wouldn't. However, I might have to test with bigger images